source/non_matching/coupling.cc

```diff
@@ -40,6 +40,72 @@
 DEAL_II_NAMESPACE_OPEN
 namespace NonMatching
 {
+  namespace internal
+  {
+    template <int dim0, int dim1, int spacedim>
+    std::pair<std::vector<Point<spacedim>>, std::vector<unsigned int>>
+    compute_used_cells(const GridTools::Cache<dim0, spacedim> &cache,
```

Comment:
I don't find the name particularly well chosen -- "used" by what? There should probably also be documentation as to what the function does.